std_msgs/Header header

float32 left_cx_down 
float32 left_cx_upper
float32 left_cy_down 
float32 letf_cy_upper

float32 mid_cx_down
float32 mid_cx_upper 
float32 mid_cy_down 
float32 mid_cy_upper 

float32 rig_cx_down 
float32 rig_cx_upper 
float32 rig_cy_down 
float32 rig_cy_upper 